parser/context.go

```diff
@@ -35,5 +37,6 @@ func NewContext(
 		Database:       db,
 		Modules:        modules,
 		Logger:         logger,
+		Prometheus:     prometheus.NewServer(config.Monitoring.Port),
```

Comment:
Wouldn't this case an error if `config.Monitoring` is null (i.e. it was not sent inside the config? I think we should just return a `nil` Prometheus instead. We could just pass `config.Monitoring` to the `NewServer` method and return a `nil` instance if the monitoring is either disabled or the config is not set at all. Then, inside the main we just call `.Start` with a `nil` check and if the server is `nil` then we do nothing. What do you think?

Reply:
Yes, I was forcing the definition of the `monitoring` section inside the config. We can make it optional, as you suggest, to maintain backward compatibility.